internal/support/notifications/application/send.go

```
@@ -112,7 +112,7 @@ func escalatedSend(dic *di.Container, n models.Notification, trans models.Transm
 	}
 
 	for _, address := range sub.Channels {
-		go transmit(dic, escalated, sub, address)
+		go transmit(dic, escalated, sub, address) // nolint:errcheck
```

Seems clearer to just suppress the error.